common-lib/src/main/scala/com/gu/mediaservice/lib/imaging/ImageOperations.scala

```diff
@@ -137,19 +165,23 @@ class ImageOperations(playPath: String) extends GridLogging {
                       qual: Double = 100d,
                       outputFile: File,
                       iccColourSpace: Option[String],
-                      colourModel: Option[String])(implicit logMarker: LogMarker): Future[(File, MimeType)] = {
+                      colourModel: Option[String],
+                      isTransformedFromSource: Boolean
```

Comment:
Let's add a comment to document the new param.

Reply:
Refactored w/ @andrew-nowak and @paperboyo to pass a `BrowserViewable` image straight through, negating the need for a comment 🎉